components/UI/RangeHistogram/components/Axis.tsx

```diff
@@ -47,8 +48,8 @@ export function Axis({ data, range, onSelect }: Props) {
                 <div
                     aria-hidden
                     key={item.value}
-                    onClick={() => onClick?.(item.value)}
-                    onKeyUp={() => onClick?.(item.value)}
+                    // onClick={() => onClick?.(item.value)}
```

Comment:
maybe we can return this behaviour?

Reply:
i would prefer not to return it because we already have two approaches of selection